README.md

```diff
@@ -96,28 +93,27 @@ jobs:
-    duration value, such as "1m" or "5s". The value must be between "1m" and
-    "10m".
+-   <a name="scan_timeout"></a><a href="#user-content-scan_timeout"><code>scan_timeout</code></a>: _(Optional, default: `3m`)_ The maximum time before the scanning stops. The value must be between `1m` and
+    `10m`. The default is `3m`.
 
--   <a name="ignore_violations"></a><a href="#user-content-ignore_violations"><code>ignore_violations</code></a>: _(Optional)_ Whether violations found in IaC file should be ignored when determining
-    the build status. This input does not apply to violations that are related
-    to generating SARIF reports and determining the `iac_scan_result`.
+-   <a name="ignore_violations"></a><a href="#user-content-ignore_violations"><code>ignore_violations</code></a>: _(Optional)_ Whether violations found in IaC file should be ignored when determining the build status. This input doesn’t apply to  violations that are related to generating SARIF reports and determining the `iac_scan_result`. The default is `false`.
 
--   <a name="failure_criteria"></a><a href="#user-content-failure_criteria"><code>failure_criteria</code></a>: _(Optional, default: `Critical:1, High:1, Medium:1, Low:1, Operator:OR`)_ Ffailure criteria that determines the workflow build status. You can set a
+-   <a name="failure_criteria"></a><a href="#user-content-failure_criteria"><code>failure_criteria</code></a>: _(Optional, default: `Critical:1, High:1, Medium:1, Low:1, Operator:OR`)_ The failure criteria that determines the workflow build status. You can set a
     threshold for the number of critical, high, medium, and low severity
-    issues and use an aggregator (either `and` or `or`) to evaluate the
+    issues and use an aggregator (either `AND` or `OR`) to evaluate the
     criteria.
 
     To determine whether a build has failed, the threshold for each severity
     is evaluated against the count of issues with that severity in the IaC
     scan results and then severity level evaluations are aggregated using
-    `AND` or `OR` to arrive at `failure_criteria` value.
+    `AND` or `OR` to arrive at `failure_criteria` value. You must include an aggregator in the string. The aggregator value is case-sensitive.
+
+    For example, if you set the failure criteria to `HIGH:1,LOW:1,OPERATOR:OR`, the workflow fails if there is 1 or more HIGH severity findings <strong>or</strong> 1 or more LOW severity findings. If you set the failure criteria to `HIGH:1,LOW:1,OPERATOR:AND`, the workflow fails if there is 1 or more HIGH severity findings <strong>and</strong> 1 or more LOW severity findings.
 
     If the `failure_criteria` evaluates to `true`, the workflow is marked as
-    `FAILED`. Otherwise, the workflow is marked as `SUCCESS`.
+    `FAILED`. Otherwise, the workflow is marked as `SUCCESS`. The default is 'Critical:1,High:1,Medium:1,Low:1,Operator:OR'.
```

Comment:
These are auto-generated from https://github.com/google-github-actions/analyze-code-security-scc/blob/main/action.yml

Reply:
Updated the PR as per your suggestions.